programs/clearing_house/src/math/auction.rs

```diff
@@ -55,6 +58,26 @@ pub fn calculate_auction_end_price(
     Ok(auction_end_price)
 }
 
+pub fn calculate_spot_auction_end_price(
+    oracle_price: &OraclePriceData,
+    direction: PositionDirection,
+) -> ClearingHouseResult<u128> {
+    let numerator = match direction {
+        PositionDirection::Long => {
+            BID_ASK_SPREAD_PRECISION + BID_ASK_SPREAD_PRECISION / 100 // 1%
```

Comment:
can we put this `1%` value on the bank?

Reply:
eh.. honestly it doesnt really matter since this is just for when end price isnt specified on the market order